src/main/java/ui/UI.java

```diff
@@ -143,6 +143,11 @@ private void login(boolean isBypassLogin) {
         getMainWindowHandle(mainStage.getTitle());
     }
 
+    private void openAndSaveSampleBoard(){
```

Comment:
Is this the correct name for this method? The code is all about 'creating' the board, not opening or saving.